nginx_config_reloader/__init__.py

```diff
@@ -45,10 +45,20 @@
 # - also takes into account double slashes
 # Because of bash escaping problems we define quote's in octal format \042 == ' and \047 == "
 ILLEGAL_INCLUDE_REGEX = "^(?!\s*#)\s*(include|load_module)\s*" \
-                        "(\\042|\\047)?" \
+                        "(\\042|\\047)?\s*" \
                         "(?=.*\.\.|/+etc/+nginx/+app_bak|/+(?!etc/+nginx))" \
-                        "(\\042|\\047)?"
-
+                        "(\\042|\\047)?\s*"
+
+# For security reasons the following nginx configuration parameters are forbidden
+FORBIDDEN_CONFIG_REGEX = \
+    [
+        ("client_body_temp_path", "Usage of configuration parameter client_body_temp_path is not allowed."),
+        ("^(?!\s*#)\s*(access|error)_log\s*"
+         "(\\042|\\047)?\s*"
+         "(?!(off|on|/+data/+))(?=.*\.\.|/+(?!data)|\w)"
+         "(\\042|\\047)?\s*",
+         "It's not allowed store access_log or error_log outside of /data/.")
+     ]
```

Comment:
this is the reason why it didn't seem feasible:
`if not self.allow_includes:`
https://github.com/ByteInternet/nginx_config_reloader/pull/12/files/57ed3ec650c1aed51f57681468f4dae74dfc4203#diff-19646a2bc74521520988231c63aa9f8bR147

I don't think this tool should allow out of place includes breaking the atomicity of applying new configs. If we ditch that option it'll save us a few lines of code.

Couldn't come up with a good implementation without removing that flag.
Should we remove it?